lang/swi-prolog-devel/Portfile

```diff
@@ -1,9 +1,10 @@
-PortSystem	1.0
+PortSystem    1.0
+PortGroup     cmake 1.1
```

Comment:
FYI, indentation seems wrong here. I'm not going to hold up committing this for that, but you should fix it in the future. You should also add a modeline per our standards.